test_idref.py

```diff
@@ -0,0 +1,86 @@
+from invenio_db import db
```

Comment:
copyright is missing but are you sure that this file should be a part of this commit? If yes, it should be either a   cli script or a test function in the `tests` directory.

Reply:
Where does that come from?